DirectProgramming/DPC++FPGA/Tutorials/Tools/experimental/platform-designer/add-quartus/system_console/test.bat

```diff
@@ -0,0 +1,22 @@
+@ REM Check if quartus.exe is on the PATH
```

Comment:
No Linux test.sh?

Reply:
most users who run this on hardware will use a work laptop connected to a board through a USB cable. this script is simple enough that a linux person can re-build it themselves, but I don't have a convenient environment to test in.